src/System.Security.Cryptography.Cng/tests/Configurations.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netfx-Windows_NT;
```

Comment:
Looks like it is not in standard yet: https://github.com/dotnet/standard/blob/master/netstandard/ref/mscorlib.cs#L12105-L12116 

@weshaggard perhaps we should fix that first and close this PR for now until that gets fixed?